amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/Lease.java

```diff
@@ -231,6 +246,16 @@ public void parentShardIds(@NonNull final Collection<String> parentShardIds) {
         this.parentShardIds.addAll(parentShardIds);
     }
 
+    /**
+     * Sets childShardIds.
+     *
+     * @param childShardIds may not be null
+     */
+    public void childShardIds(@NonNull final Collection<String> childShardIds) {
+        this.childShardIds.clear();
```

Comment:
Why do we need to do this if childShardIds is a set?

Reply:
Will remove